easybuild/easyconfigs/b/BlobTools/BlobTools-20180528-foss-2018b-Python-2.7.15.eb

```diff
@@ -11,12 +11,23 @@ description = """ A modular command-line solution for visualisation,
 
 toolchain = {'name': 'foss', 'version': '2018b'}
 
-source_urls = ['https://github.com/drl/blobtools/archive/']
-sources = [{'download_filename': '%s.tar.gz' % commit, 'filename': SOURCE_TAR_GZ}]
+source_urls = [
+    'https://github.com/drl/blobtools/archive/',
+    'ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/',
+]
+sources = [
+    {
+        'download_filename': '%s.tar.gz' % commit,
+        'filename': SOURCE_TAR_GZ,
+        'extract_cmd': "tar -xzf %s -C %(builddir)s --strip-components 1",
+    },
+    {'filename': 'taxdump.tar.gz', 'extract_cmd': "cp %s %(builddir)s/data/"},
```

Comment:
@smoors We should version this file on download (with a datestamp), since it's bound to change overtime (and hence so will the checksum), or download from ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump_archive/ instead?

Reply:
@smoors ping!